assignment-5/deleteApp.js

```diff
@@ -0,0 +1,15 @@
+module.exports.deleteApp = function (app, user) {
```

Comment:
@JainVayam Your delete user logic is absolutely wrong. Did you check the code before creating the PR?